src/pybind/mgr/dashboard_v2/module.py

```diff
@@ -46,6 +50,7 @@ class Module(MgrModule):
     def __init__(self, *args, **kwargs):
         super(Module, self).__init__(*args, **kwargs)
         logger.logger = self._logger
+        self.url_prefix = ''
```

Comment:
Consider to transfor this into a property, thus it is read-only.

```
@property
def url_prefix(self):
    return self._url_prefix
```

Reply:
Or
```
@property
def url_prefix(self):
    url_prefix = urljoin('/', self.get_config('url_prefix', default=''))
    return url_prefix.rstrip('/')
```